ocs_ci/ocs/cluster.py

```diff
@@ -2644,6 +2654,127 @@ def exec_cmd_on_cluster_node(self, cmd):
         return_stdout = return_output[1]
         return return_stdout
 
+    def get_thin_provisioning_alerts(self):
+        """
+        Get the list of alerts that active in the cluster
+
+        Returns:
+            list: alrets name
+
+        """
+        if not isinstance(self.new_prom, PrometheusAPI):
+            self.init_prom()
+
+        alert_full = self.new_prom.get("alerts")
+        alerts_data = alert_full.json().get("data").get("alerts")
+        alerts_names = list()
+        for entity in alerts_data:
+            logger.debug(entity.get("labels").get("alertname"))
+            alerts_names.append(entity.get("labels").get("alertname"))
+
+        return alerts_names
+
+    def check_for_alert(self, alert_name):
+        """
+        Check to see if a given alert is available
+
+        Args:
+            alert_name (str): Alert name
+
+        Returns:
+            bool: True if alert is available else False
+
+        """
+        if alert_name in self.get_thin_provisioning_alerts():
+            return True
+
+        return False
+
+    def parse_topolvm_metrics(self, metrics):
+        """
+        Returns the name and value of topolvm metrics
+
+        Args:
+            metric_name (list): metrics name to be paesed
+
+        Returns:
+            dict: topolvm metrics by: names: value
+        """
+        if not isinstance(self.new_prom, PrometheusAPI):
+            self.init_prom()
+
+        metrics_short = dict()
+        for metric_name in metrics:
+            metric_full = self.new_prom.query(metric_name)
+            metric_value = metric_full[0].get("value")[1]
+            logger.info(f"metric: {metric_name} : {metric_value}")
+            metrics_short[metric_name] = metric_value
+
+        return metrics_short
+
+    def validate_metric_of_thin_pool_data_percent(self):
```

Comment:
Can we have this function more general?
Like validate_metric(mertic, expected_value) ? 
So we don't have function for every metric.

Reply:
we can, but the validation is not the same, so it would be multiple "if" condition. not sure it is better

Reply:
Do you calculation what is expected in the test, then send the expected value. Why have condition in the function? Can you elaborate?